proto/relay/ethereum_light_client/config/config.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package relayer.provers.ethereum_light_client.config;
+package relay.ethereum_light_client.config;
```

Comment:
Keep the package name as original and move this file to `relay/provers/ethereum_light_client/config` directory

Reply:
Revert package name 
https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/3/commits/7484802a5ca3f65de988f15af8808db21dce92be